modules/brushingandlinking/include/modules/brushingandlinking/brushingandlinkingmanager.h

```diff
@@ -65,11 +65,14 @@ class IVW_MODULE_BRUSHINGANDLINKING_API BrushingAndLinkingManager {
     bool isColumnSelected(size_t column) const;
 
     void setSelected(const BrushingAndLinkingInport* src, const std::unordered_set<size_t>& idx);
+    void clearSelected();
```

Comment:
what's the thinking here? why not use a empty set together with setSelected? How will this keep track of the sources?

Reply:
should this be public? a processor using the manager should not call this right?

Reply:
This was based on the fact that none of the set functions in the manager actually consider the source. This is only of importance for the events, not the manager.
https://github.com/inviwo/inviwo/blob/7ba4f7571fc2d0a3a2957cc2f6ad3ba23e6684e1/modules/brushingandlinking/src/brushingandlinkingmanager.cpp#L63-L75

re "public", that was the idea.

Reply:
the filter does consider source...

Reply:
However, the source is only used as key in the unordered map which is going to be cleared.

Reply:
But will all the sources of filtering get updated when cleared like this

Reply:
I assume so. Maybe @r-englund can shed some light on this.

Reply:
Is the filtering comes from the pcp plot for example, from each axis range. How is the pcp plot supposed to know that is should reset all the axis ranges when this happens?!? I think this is quite broken now

Reply:
What are the use of clear? Who/what is calling it?